pkg/network/node/ovscontroller.go

```diff
@@ -46,6 +48,41 @@ func (oc *ovsController) getVersionNote() string {
 	return fmt.Sprintf("%02X.%02X", oc.pluginId, ruleVersion)
 }
 
+func (oc *ovsController) updateOVSMetrics() {
+	flows, err := oc.ovs.DumpFlows()
+	if err == nil {
+		OVSFlows.Set(float64(len(flows)))
```

Comment:
if we only care about the *number* of flows it seems excessive to call DumpFlows every time... we ought to be able to keep track of this by hand...